src/Arcus.Messaging.Pumps.Abstractions/Resiliency/IMessagePumpCircuitBreaker.cs

```diff
@@ -16,5 +16,12 @@ public interface IMessagePumpCircuitBreaker
         /// <param name="configureOptions">The optional user-configurable options to manipulate the workings of the message pump interaction.</param>
         /// <exception cref="ArgumentException">Thrown when the <paramref name="jobId"/> is blank.</exception>
         Task PauseMessageProcessingAsync(string jobId, Action<MessagePumpCircuitBreakerOptions> configureOptions);
+
```

Comment:
The interface should have a property that exposes the state of the CircuitBreaker (open, closed, halfopen).

Reply:
Done with the `GetMessageProcessingState` on the circuit breaker.